src/ckb_chain/mod.rs

```diff
@@ -0,0 +1,9 @@
+mod actor;
```

Comment:
The functionality of the mod `ckb_chain` is indefensible for the `ckb` payment channel network. I think this create would suit itself better as an sub module to `ckb`. How do you think of that? One benefit of using `ckb_chain` as a separate service is that in this case we can run a standalone daemon remotely that does nothing but signing funding transactions. I don't know if this is a real use case.

Reply:
I have no strong opinion on this. However, I think we'd better refactor it into a submodule later after the demo.